apps/web/src/pages/WorkforceAdjustment/WorkforceAdjustmentPage.tsx

```diff
@@ -0,0 +1,620 @@
+              }) + intl.formatMessage(commonMessages.dividingColon)}
+            </p>
+            <Ul space="sm">
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Your job falls under one of the <strong>functional communities</strong> supported by this platform",
+                  id: "vKVb4V",
+                  description:
+                    "Indicator one, functional communities use these tools",
+                })}
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You have a <strong>term position</strong> that isn't being renewed or is being terminated early",
+                  id: "qWZblr",
+                  description:
+                    "Indicator two, your term position is not being renewed",
+                })}
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You're an indeterminate employee who has received a <strong>workforce adjustment letter</strong>",
+                  id: "wJiBDR",
+                  description: "Indicator three, you received WFA letter",
+                })}
+              </li>
+              <li>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You're an indeterminate employee interested in <strong>a package to leave the public service voluntarily</strong>",
+                  id: "dF2qSb",
+                  description:
+                    "Indicator four, you are interested in a package",
+                })}
+              </li>
+            </Ul>
+          </div>
+        </div>
+        <Heading color="primary" icon={UserGroupIcon} className="font-normal">
+          {intl.formatMessage({
+            defaultMessage: "Employees supported by this service",
+            id: "c2pcX9",
+            description:
+              "Title for users supported by workforce adjustment tools",
+          })}
+        </Heading>
+        <p className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "At this time, our services are available for employees whose current job falls under one of the functional communities supported by this platform. Basically, some functional community offices manage talent through this platform, and they won’t have roles to offer employees outside the scope of their mandate areas. (For those in other functional communities, we encourage you to explore options through GCX link.)",
+            id: "s2LoqF",
+            description:
+              "Paragraph describing the users who are supported by the workforce adjustment tools",
+          })}
+        </p>
+        <div className="grid gap-6 sm:grid-cols-2 md:grid-cols-3">
+          <FeatureBlock
+            content={{
```

Comment:
The Figma has them as white background 

Figma

<img width="828" height="165" alt="image" src="https://github.com/user-attachments/assets/33fcabb8-590b-4e3d-95bf-bc1755cf95ce" />

Current

<img width="858" height="288" alt="image" src="https://github.com/user-attachments/assets/1d6f41c2-972b-4c7d-9a6b-b76bda3d40b9" />